### ISSUE TITLE
Shall I use cmu or vgg if I want to train the original paper's model

### ISSUE THREAD

**Issue by Aileenlingyu** [ildoonet/tf-pose-estimation#145]
Hi! I was wondering that where are openpose_vgg16.npy and openpose_coco.npy from? Are they just pretrained weights on ImageNet, which we can use for fine-tuning? Or they are the already trained model for openpose? If so, what is the difference between them?

I just started to train the original paper's model, I trained with --model=cmu and used pretrained weights openpose_coco.npy. I thought this pretrained weights file is the pretrained weight on ImageNet, which we use for fine-tuning. But I found that the loss looks good at the very beginning, the first validation loss is around 30, I was wondering that, is openpose_coco.npy already trained by author? Shall I use openpose_vgg16.npy instead?

Also It seems that in the original paper they are using vgg19, is that also a difference?

Thanks a lot for any hints!

**Comment by JakubWojciechowski** [comment]
Hello! Models of the vgg and cmu networks are identical, they differ only in the loaded weights. VGG loads pre-trained weights only for the base network (without Convolutional Pose Machines). In CMU model all layers are pre-trained, it is already trained model for openpose.

**Comment by Aileenlingyu** [comment]
Thanks so much! @JakubWojciechowski

**Comment by GajjarMihir** [comment]
@Aileenlingyu I am also trying to train the model. Can you please share how much time it takes to train the model on what hardware? and can you also share your accuracy?

**Comment by Aileenlingyu** [comment]
@GajjarMihir Sure, it takes around 12 days to run 149000 iterations and the loss reaches around 31. (This is the last layer validation loss, which is the average of PAF and joint heatmap loss). The GPU card is Tesla M40.

**Comment by GajjarMihir** [comment]
Thank you @Aileenlingyu . Can you please share the epoch, batch size and details of other hyperparameters? And did you find the testing accuracy in terms of mean Average precision (AP) (which is the evaluation metric for COCO dataset)?

**Comment by Aileenlingyu** [comment]
Hi @GajjarMihir , I didn't change hyper-parameters for optimizer, the input resolution is 368x368, I used learning rate of 0.0001 which is the recommended value from the demo author, considering the GPU memory, I used batch size of 16. I haven't tested the model using mAP on COCO.

**Comment by jzwang123** [comment]
> Hi @GajjarMihir , I didn't change hyper-parameters for optimizer, the input resolution is 368x368, I used learning rate of 0.0001 which is the recommended value from the demo author, considering the GPU memory, I used batch size of 16. I haven't tested the model using mAP on COCO.

Hi @Aileenlingyu , did you have a chance to test mAP on COCO later on?
